lib/orderbook/OrderBook.ts

```diff
@@ -51,10 +53,14 @@ class OrderBook extends EventEmitter {
     this.repository = new OrderBookRepository(models);
     if (pool) {
       pool.on('packet.order', this.addPeerOrder);
-      pool.on('packet.orderInvalidation', body => this.removePeerOrder(body.orderId, body.pairId)); // TODO: implement quantity invalidation
+      pool.on('packet.orderInvalidation', (body: orders.OrderIdentifier) => {
+        // TODO: implement quantity invalidation
+        if (this.removePeerOrder(body.orderId, body.pairId)) {
+          this.emit('peerOrder.invalidation', body);
```

Comment:
It would be better to emit from `removePeerOrder`. 
Mainly because that's how you did it on the batch removal (`removePeerOrders`), but also to disallow removing without emitting.

Reply:
I will fix that in #266 because I had to resolve conflicts in this line anyway.